src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    protected LocalDate date;
+
+    /**
+     * The constructor for an Event object.
+     *
+     * @param description The description of the Event.
+     * @param date The date of the Event.
+     */
+    public Event(String description, LocalDate date) {
+        super(description);
+        this.date = date;
+        this.taskType = TaskType.E;
+    }
+
+    @Override
+    public String toString() {
+        String str = super.toString();
+        str += String.format(" (at: %s)", this.date.format(DateTimeFormatter.ofPattern("dd MMM yyyy")));
+        return str;
+    }
+
+    @Override
```

Comment:
May I recommend that you call super.getDataString() instead

Reply:
Thanks for the suggestion! Made my code much cleaner!! (: